helm-chart/ray-cluster/values.yaml

```diff
@@ -84,13 +84,14 @@ head:
   affinity: {}
   # Ray container security context.
   securityContext: {}
-  volumes:
-    - name: log-volume
-      emptyDir: {}
-  # Ray writes logs to /tmp/ray/session_latests/logs
-  volumeMounts:
-    - mountPath: /tmp/ray
-      name: log-volume
+  # Optional log and log volumeMounts
```

Comment:
They are optional but make for a good default, so keeping them the same would be better.